lib/rubocop/cop/lint/numeric_operation_with_constant_result.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # Certain numeric operations have a constant result, usually 0 or 1.
```

Comment:
You should probably mention the other cop that's a relative to this one here, so people understand well why the scope is so narrow.